src/setfit/modeling.py

```diff
@@ -132,16 +132,17 @@ def forward(
 
         return outputs
 
-    def predict_proba(self, x_test: torch.Tensor) -> torch.Tensor:
+    def predict_proba(self, x_test: Union[torch.Tensor, "ndarray"]) -> torch.Tensor:
         self.eval()
 
-        return self(x_test)
-
-    def predict(self, x_test: Union[torch.Tensor, "ndarray"]) -> Union[torch.Tensor, "ndarray"]:
         is_tensor = isinstance(x_test, torch.Tensor)
         if not is_tensor:  # then assume it's ndarray
             x_test = torch.Tensor(x_test).to(self.device)
 
+        return self(x_test)
```

Comment:
As above maybe we can convert the output to `ndarray` if the input is a `ndarray`? Specifically,
```suggestion
        out = self(x_test)
        if not is_tensor:
            return out.cpu().numpy()
    
        return out
```